config.yaml

```diff
@@ -36,3 +36,13 @@ options:
     default: true
     type: boolean
     description: Enable opensearch-knn
+
+  profile:
+    type: string
+    default: "production"
```

Comment:
@Mehdi-Bendriss default should be "production"?

Reply:
seems like it from [the spec](https://docs.google.com/document/d/1bq7KyUAPIIcUWGgifW05p40v7lGXs3PhMznM2ECaMKY/edit?tab=t.0)